### PR TITLE
Add README.md of kubernetes show logs command (#226)

### PR DESCRIPTION
**Pull request by Valeyard1**
# Pull Request

## What I did
Add an explanation of how to run the command `rit kubernetes show logs`.

## How I did it
After reading the `config.json` file of this formula I figure it out which parameters to pass to the command via stdin.

## What kind of change does this PR make

- [ ] Major
- [X] Minor
- [ ] Patch

## Description for the changelog
The command `rit kubernetes show logs` now have a README.md explaning how to use it.

### CODE REVIEW

**Comment by Valeyard1** [comment]
Some notes:
- I wrote the wrong number in the commit, I linked to the issue 228 instead of 226, I fixed it after a force push
- I uploaded the GIF with [asciinema](https://asciinema.org), it's a program which can record the terminal, if it's mandatory to record it with the gif file format, I can record it again

File: kubernetes/show/logs/README.md [hunk 1]
```diff
@@ -1,11 +1,35 @@
 # Kubernetes show logs
 
-## command
+## Promises
```
Comment (GuillaumeFalourd) [inline kubernetes/show/logs/README.md:3]: I believe you meant "Premisses", right ?

**Comment by GuillaumeFalourd** [comment]
Hi @Valeyard1 , thank you for contributing 👍 

I suggested one update, let me know if it makes sense 😄

File: kubernetes/show/logs/README.md (hunk 1 above)
Comment (Valeyard1) [reply kubernetes/show/logs/README.md]: Yes, I'm sorry

Comment (GuillaumeFalourd) [reply kubernetes/show/logs/README.md]: No problem 👌

**Review by GuillaumeFalourd — APPROVED** [review]

**Comment by GuillaumeFalourd** [comment]
Great, well done @Valeyard1 🚀  

Have you contributed to another open source project from ZupIT repository ?

If you have 2 or more PR you'll be able enter the https://insights.zup.com.br/hacktoberfest challenge 😄 

Let me know 👍

**Pull request merged**